src/__tests__/tigris.filters.spec.ts

```diff
@@ -154,6 +154,44 @@ describe("filters tests", () => {
 		expect(Utility.filterToString(tigrisFilter)).toBe('{"balance":{"$gte":10}}');
 	});
 
+	it("not Filter", () => {
+		const tigrisFilter: Filter<Student> = {
+			name: {
+				$not: "Jack",
+			},
+		};
+		expect(Utility.filterToString(tigrisFilter)).toBe('{"name":{"$not":"Jack"}}');
+	});
+
+	it("contains Filter(string)", () => {
+		const tigrisFilter: Filter<Student> = {
+			name: {
+				$contains: "Adam",
+			},
+		};
+		expect(Utility.filterToString(tigrisFilter)).toBe('{"name":{"$contains":"Adam"}}');
+	});
+
+	it("contains Filter(string[])", () => {
+		const tigrisFilter: Filter<Student> = {
+			name: {
+				$contains: ["Adam", "Warlock", "Steven"],
```

Comment:
Can we restrict it to only allow strings instead of an array of strings?

Reply:
@himank sure.

@himank I was relying on the issue description specified in #325 by @ovaistariq
And I thought the server supported both payloads?

```
filter: {
op: "$contains",
employeeName: ["Jack", "Tim", "Sarah"]
}
```

I've update the PR accordingly

Reply:
My bad. I have corrected the spec